tests/functionaltests/FunctionalTestHelpers.mm

```diff
@@ -30,11 +30,7 @@
 
 // Launches the functional test app
 bool FunctionalTestSetupUIApplication() {
-    RunSynchronouslyOnMainThread(^{
-        // The name of our default 'AppDelegate' class
-        UIApplicationInitializeFunctionalTest(nullptr, Strings::NarrowToWide<std::wstring>(NSStringFromClass([AppDelegate class])).c_str());
-    });
-
+    UIApplicationInitializeFunctionalTest(nullptr, Strings::NarrowToWide<std::wstring>(NSStringFromClass([AppDelegate class])).c_str());
```

Comment:
Is that what those // are for?
Will do.

---
In reply to: [100440979](https://github.com/Microsoft/WinObjC/pull/1937#discussion_r100440979) [](ancestors = 100440979)